hack/push-helm.sh

```diff
@@ -31,4 +31,4 @@ yq -i "$image_ref |= \"$IMG\"" "$chart_dir/values.yaml"
 yq -i ".name |= \"$name\"" "$chart_dir/Chart.yaml"
 
 helm package "$chart_dir" -d "$chart_dir" --version "$tag"
-helm push "$chart_dir/$name-$tag.tgz" "oci://$registry"
+helm push --plain-http "$chart_dir/$name-$tag.tgz" "oci://$registry"
```

Comment:
This doesn't seem to work as expected. See the errors in the tests ([ref](https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_gardener/10257/pull-gardener-e2e-kind-operator/1835927403969384448#1:build-log.txt%3A210-211), [ref](https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_gardener/10257/pull-gardener-e2e-kind-operator-seed/1835927403130523648#1:build-log.txt%3A211))

Reply:
Hi Oliver, it turns out I had forgotten to add `garden.local.gardener.cloud:5001` as an insecure registry to the other two skaffold pipeline files. There's one last issue I'm trying to figure out and then I'll clean up this PR for merging